src/chartsFactory.jsx

```diff
@@ -33,9 +30,7 @@ module.exports = function (chartType, Highcharts){
     },
 
     shouldComponentUpdate(nextProps) {
-      // if isPureConfig is explicitly false, don't bother comparing configs, otherwise compare if not explicitly true
-      if (this.props.isPureConfig === false
-          || (this.props.isPureConfig !== true && !(this.props.config === nextProps.config))) {
+      if (!this.props.neverReflow && (!this.props.isPureConfig || !(this.props.config === nextProps.config))) {
```

Comment:
nitpick, but can you negate the whole thing? So it looks like below:
```javascript
if (this.props.neverReflow || (this.props.isPureConfig  && this.props.config === nextProps.config)) {
  return true;
}
this.renderChart(nextProps.config);
```
This way we won't have negation for every predicate.